docs/reference/hooks/index.md

```diff
@@ -0,0 +1,564 @@
+| hooks                                 | Object whose keys must match the hook names outlined in the above [Hooks Specification](#specification). Arguments can be provided within this string by separating them with spaces.                                                                                                                                                                                                                           | Required |
+| config                                | Object of key-value settings.                                                                                                                                                                                                                                                                                                                                                                                   | Optional |
+| config.protocol-version               | Array of strings representing the named CLI-SDK protocols supported by the SDK, in descending order of support, as in the first element in the array defines the preferred protocol for use by the SDK, the second element defines the next-preferred protocol, and so on. The only supported named protocol currently is `message-boundaries`. The CLI will use the v1 protocol if this field is not provided. | Optional |
+| config.watch                          | Object with configuration settings for file-watching.                                                                                                                                                                                                                                                                                                                                                           | Optional |
+| config.sdk-managed-connection-enabled | Boolean specifying whether the WebSocket connection between the CLI and Slack should be managed by the CLI or by the SDK during `slack run` executions. If `true`, the SDK will manage this connection. If `false` or not provided, the CLI will manage this connection.                                                                                                                                        | Optional |
+| config.trigger-paths                  | Array of strings that are paths to files of trigger definitions.                                                                                                                                                                                                                                                                                                                                                | Optional |
```

Comment:
📚 note: This reference was added in recent commit https://github.com/slackapi/slack-cli/pull/174/commits/80689d9568625a5fc6192290fc1c8e8573aec7b1 but I think separate reference for the `config` options might be nice to follow up with.

👁️‍🗨️ note: Right now this option is also documented at: https://docs.slack.dev/tools/deno-slack-sdk/guides/using-triggers/#custom-path